docs/en-US/component/select-v2.md

```diff
@@ -203,27 +205,27 @@ select-v2/props
 | loading-text | 远程加载时显示的文字 | string | — | 加载中 | -->
 </span>
 
-## props
+#### props
```

Comment:
Maybe the `props` level just needs to be set to `### props`?

Reply:
i agree